iot/iot-jdbc-base/src/main/java/io/enmasse/iot/jdbc/store/StatementConfiguration.java

```diff
@@ -159,12 +159,7 @@ public void dump(final Logger logger) {
      * @return A new YAML parser, never returns {@code null}.
      */
     static Yaml createYamlParser() {
-        return new Yaml(new Constructor() {
-            @Override
-            protected Class<?> getClassForName(String name) throws ClassNotFoundException {
-                throw new IllegalArgumentException("Class instantiation is disabled");
-            }
-        });
+        return new Yaml(new Constructor(Map.class));
```

Comment:
This doesn't prevent the !!class being constructed.

Reply:
It did, but as you mentioned, only on the root level. I switched to loading the YAML via `load` (vs `loadAs(Map.class)`.

With `loadAs(Map.class)` the problem is, that the constructor still tries to construct an new object, by class name, for Map.

`load` on the other hand doesn't do that, as it returns basic types only, not mapping anything to a bean. So it will only return a `Map<String,Object>`, and that allows to disable all class loading.

I also added you suggested test, which triggered the issue.